mpas_analysis/ocean/variable_stream_map.py

```diff
@@ -0,0 +1,46 @@
+# mappings of stream names from various MPAS-O versions to those in
+# mpas_analysis
+oceanStreamMap = {'timeSeriesStats': ['timeSeriesStatsOutput',
+                                      'timeSeriesStatsMonthly',
+                                      'timeSeriesStatsMonthlyOutput']}
+
+
+oceanVariableMap = {}
```

Comment:
The reason I think they should be ordered is that we should try all the variable names from the newest model commits first.  We get a random order from dict in general that is almost nearly guaranteed to be inefficient because we are likely to try elements that don't apply to the commit.  Again, not a big deal, but I brought it up because we originally talked about the implementation in terms of `OrderedDict`.